batchspawner/batchspawner.py

```diff
@@ -88,89 +88,98 @@ class BatchSpawnerBase(Spawner):
     start_timeout = Integer(300).tag(config=True)
 
     # override default server ip since batch jobs normally running remotely
-    ip = Unicode("0.0.0.0", help="Address for singleuser server to listen at").tag(config=True)
+    ip = Unicode("0.0.0.0", help="Address for singleuser server to listen at").tag(
```

Comment:
This is an abomination. If black is going to insist on breaking these lines, let's add trailing commas to the argument list so that it doesn't break the `.tag(config=True)` expression in a way that obscures the method chaining being used.